todo/android/todo/src/main/java/com/mongodb/stitch/sdk/examples/todo/MainActivity.java

```diff
@@ -98,32 +101,27 @@ public void onLogin() {
         }
 
         @Override
-        public void onLogout(final String lastProvider) {
+        public void onLogout() {
             final MainActivity activity = _main.get();
 
             final List<Task<Void>> futures = new ArrayList<>();
             if (activity != null) {
                 activity._handler.removeCallbacks(activity._refresher);
 
-                switch (lastProvider) {
-                    case GoogleAuthProviderInfo.FQ_NAME:
-                        if (activity._googleApiClient != null && activity._googleApiClient.isConnected()) {
-                            final TaskCompletionSource<Void> future = new TaskCompletionSource<>();
-                            com.google.android.gms.auth.api.Auth.GoogleSignInApi.signOut(
-                                    activity._googleApiClient).setResultCallback(new ResultCallback<Status>() {
-                                @Override
-                                public void onResult(@NonNull final Status ignored) {
-                                    future.setResult(null);
-                                }
-                            });
-                            futures.add(future.getTask());
-                        }
-                        break;
-                    case FacebookAuthProviderInfo.FQ_NAME:
-                        if (activity._fbInitOnce) {
-                            LoginManager.getInstance().logOut();
+                if (activity._googleApiClient != null) {
```

Comment:
[q] How are we accessing an underscore property here? Has this been named wrong?

Reply:
Not sure what you're asking. AFAIK, the underscore in Java doesn't enforce `private`. I'm not sure who originally created this code and/or decided on the use of underscore for the private vars, but I can do a full sweep if you think it's necessary?

Reply:
Ah, I hadn't realized what was occurring here. This makes sense now. Disregard.

Reply:
Consider it disregarded, then.